app/src/main/java/sims/michael/gerritslackbot/slack/EventGroupToSlackMessageTransformer.kt

```diff
@@ -192,11 +202,4 @@ class EventGroupToSlackMessageTransformer(
                 .joinToString(delimiter).escapeHtml()
     }
 
-    private fun <K : Any?, V> Map<K?, V>.filterNotNullKeys(): Map<K, V> {
```

Comment:
Good change, but again, isolated.

I'm actually OK with all of the "clean up" changes to the existing code being a single commit. That includes this, and the change to drop the explicit type params in `sims.michael.gerritslackbot.EventGroupingTransformer`